lib/reporters/filesize-reporter.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+const colors = require('colors');
+const filesize = require('filesize');
+const Logger = require('../logger');
+const percent = require('calc-percent');
+const Table = require('cli-table');
+
+const logger = new Logger();
+
+function filesizeReporter(statsObj, logLevel) {
+	logger.updateLevel(logLevel);
+	logger.log(createTable(statsObj.src, statsObj.dest).toString());
+}
+
+function createTable(srcObj, destObj) {
+	const table = new Table(createHeaderData());
+	const typeRows = Object.keys(srcObj).map(type => createTableRow(type, srcObj[type], destObj[type]));
+	const totalRow = createTableRow('Total', calculateTotal(srcObj), calculateTotal(destObj));
+
+	[...typeRows, totalRow.map(item => colors.bold(item))]
```

Comment:
I left the pushing to the table the way it was. I think using the spread operator is making it more readable. I did however moved the `map` that makes the footer bold to the footer statement.